distribution/lib/Standard/Base/0.0.0-dev/src/System/File/Existing_File_Behavior.enso

```diff
@@ -44,7 +44,7 @@ type Existing_File_Behavior
 
        The `action` may not be run at all in case the `Error` behavior is
        selected.
-    write : File -> (Output_Stream -> Nothing) -> Nothing ! File_Error
+    write : File -> (Output_Stream -> Any) -> Any ! File_Error
```

Comment:
```suggestion
    write : File -> (Output_Stream -> Any) -> File ! File_Error
```

I think?

Reply:
I concur.

The `write` here takes an actual `action` that will perform the writing and will return its result. I want to give us flexibility to return whatever we may need from there, we may need to return some other stuff from the `action` (I imagine like, count of written rows for example), for our internal purposes.